### ISSUE TITLE
Question about the implementation of mask

### ISSUE THREAD

**Issue by WinterPan2017** [invoker-LL/WSI-finetuning#2]
Hello, I'm confused about the mask operation in reducing patches. 
In the implementation, the mask is obtained by addition of z_mask (sampling from Bernoulli distribution) and instance_prob (predict by linear layer), I think it should be mask = z_mask. could you please tell me the reason you compute the mask in this way? Thanks a lot. 
https://github.com/invoker-LL/WSI-finetuning/blob/966290b76ca7b72af32165ebada38c06da0ff04f/models/vib_clam_wsi.py#L154

**Comment by invoker-LL** [comment]
It is similar to [reparameterization in VAE](https://github.com/AntixK/PyTorch-VAE/blob/a6896b944c918dd7030e7d795a8c13e5c6345ec7/models/beta_vae.py#L112), but recently I find that it is not necessary. 
We can also generate z_mask by other attribution methods like attention-score or grad-cam, which is more simple for understanding, and I will update this with step-2&3 recently.
